fakegripper/gripper.go

```diff
@@ -50,56 +51,32 @@ func newFakeGripper(ctx context.Context, deps resource.Dependencies, conf resour
 	return f, nil
 }
 
-func (f *fake) simulateMove(ctx context.Context, extra map[string]interface{}) {
+func (f *fake) simulateMove(ctx context.Context, extra map[string]interface{}) error {
```

Comment:
hmm, I guess we never needed these.
```suggestion
func (f *fake) simulateMove() error {
```